pytrnsys_process/converter.py

```diff
@@ -57,26 +58,26 @@ def convert_sim_results_to_csv(
             if not input_file.is_file():
```

Comment:
if input_file.is_dir()

Otherwise, try to ensure this is taken care of in the list comprehension above.